code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -293,6 +298,8 @@ the implant may become unstable and either pre-maturely inject the subject or si
 	to_chat(H, "<span class = 'notice'>You feel a surge of loyalty towards Nanotrasen.</span>")
 	return 1
 
+
+//Greytide Implant
```

Comment:
All of these comments are superfluous.  Comments should explain code.  I immediately know that this is a graytide implant because the name is right there ``` name = "Greytide Implant" ``` .  This is applicable on all of your comments naming objects that are immediately named but I'm not going to leave a comment on all of them.

Reply:
Noted: I'll fix that tomorrow.